runtime/common/src/parachains.rs

```diff
@@ -146,8 +146,18 @@ impl<Proof: Parameter + GetSessionNumber, Hash: AsRef<[u8]>> DoubleVoteReport<Pr
 			.ok_or(DoubleVoteValidityError::InvalidProof)?;
 
 		// Check signatures.
-		Self::verify_vote(&first, &self.parent_hash, &self.identity)?;
```

Comment:
The first check in this function (before `check_proof`) should be `self.proof.session_index() == self.signing_context.session_index`